euphoria-operator-testkit/src/main/java/cz/seznam/euphoria/operator/test/MapElementsTest.java

```diff
@@ -39,7 +40,7 @@ public void testOnTwoPartitions() {
       @Override
       protected Dataset<String> getOutput(Dataset<Integer> input) {
         return MapElements.of(input)
-            .using(String::valueOf)
+            .using((UnaryFunction<Integer, String>) String::valueOf)
```

Comment:
I tried to dig deeper into Java to find out what are the candidates the compiler takes into account and the result is that I don't know. Kind of magic.
```
Error:(43, 13) java: reference to using is ambiguous
  both method <OUT>using(cz.seznam.euphoria.core.client.functional.UnaryFunction<IN,OUT>) in cz.seznam.euphoria.core.client.operator.MapElements.UsingBuilder and method <OUT>using(cz.seznam.euphoria.core.client.functional.UnaryFunctionEnv<IN,OUT>) in cz.seznam.euphoria.core.client.operator.MapElements.UsingBuilder match
```